lib/commands/recordscreen.js

```diff
@@ -58,6 +58,10 @@ class ScreenRecorder {
     if (this.opts.videoScale) {
       args.push('-vf', `scale=${this.opts.videoScale}`);
     }
+    // Quicktime compatibility via pixelFormat: 'yuv420p'
+    if (this.opts.pixelFormat && this.opts.videoType === 'libx264') {
```

Comment:
I think it makes sense to simply have pixelFormat set without the additional dependency on videoType value

Reply:
In my tests, the pixelFormat had no effect on the `mjpeg` video type and it might be incompatible with video types other than `libx264`, so not sure if it's ok to unconditionally add it.

Reply:
leave it up to the client to decide that

Reply:
Done.